dbt/adapters/base/impl.py

```diff
@@ -421,7 +421,7 @@ def _get_cache_schemas(self, relation_configs: Iterable[RelationConfig]) -> Set[
         populate.
         """
         return {
-            self.Relation.create_from(quoting=self.config, relation_config=relation_config)
```

Comment:
Looks like we unintentionally removed .without_identifier as part of the adapter decoupling: https://github.com/dbt-labs/dbt-core/pull/8906/files#diff-aa1e9e602a0f1e2930b465b30a35a28778fbe131da4fbf2c4e88dd52ca44f140R422-R423

Reply:
@colin-rogers-dbt @McKnight-42 I wonder if this was the issue we were seeing in that `dbt-snowflake` test. It feels like more than a coincidence.